modules/tinymce/src/themes/silver/main/ts/api/Settings.ts

```diff
@@ -57,9 +56,8 @@ const isToolbarEnabled = (editor: Editor): boolean => {
 
 // Convert toolbar<n> into toolbars array
 const getMultipleToolbarsSetting = (editor: Editor): Option<string[]> => {
-  const keys = Obj.keys(editor.settings);
-  const toolbarKeys = Arr.filter(keys, (key) => /^toolbar([1-9])$/.test(key));
-  const toolbars = Arr.map(toolbarKeys, (key) => editor.getParam(key, false, 'string'));
+  const potentialKeys = [ 'toolbar1', 'toolbar2', 'toolbar3', 'toolbar4', 'toolbar5', 'toolbar6', 'toolbar7', 'toolbar8', 'toolbar9' ];
```

Comment:
This bloats the code a fair bit more than the previous logic. How about something like this instead?
```
const toolbars = Arr.range(9, (num) => editor.getParam('toolbar' + (num + 1), false, 'string'));
```

Reply:
That could work. I was hesitant to use that kind of logic, but if it's given the go-ahead I'm fine with it.